src/main/java/org/sunbird/profile/controller/ProfileController.java

```diff
@@ -45,4 +45,10 @@ public ResponseEntity<?> parichayProfileUpdate(@RequestBody Map<String, Object>
 		SBApiResponse response = profileService.userBasicProfileUpdate(request);
 		return new ResponseEntity<>(response, response.getResponseCode());
 	}
+	
+	@PostMapping("/user/v1/signup")
+	public ResponseEntity<?> signupUser(@RequestBody Map<String,Object> request) throws Exception{
```

Comment:
Change the method name to "userSignUp"

Reply:
update the names of the service methods as well

Reply:
changed